src/lightning_lite/lite.py

```diff
@@ -159,19 +166,77 @@ def setup(
             model = self._move_model_to_device(model=model, optimizers=list(optimizers))
 
         # Let accelerator/plugin wrap and connect the models and optimizers
-        model, optimizers = self._strategy.setup_module_and_optimizers(model, list(optimizers))
+        if optimizers:
+            model, optimizers = self._strategy.setup_module_and_optimizers(  # type: ignore[assignment]
```

Comment:
This uses `module` but the user API is `setup_model`. Use only `module`?

Reply:
Internally in the strategy we want to use "module" to differentiate between the old way of referring to "model" as the wrapped LightningModule. But the user facing API in Lite shouldn't care about that, because it never had to care about this distinction.

Reply:
I would have them match to avoid confusion from those who snoop inside. But not a strong opinion.

Reply:
I agree with @carmocca on this. Especially if lite is meant for people who want to read the code there shouldn't be confusion be these simple naming conventions from "legacy PL"

Reply:
I updated the relevant parts that this PR touches in [8220c0e](https://github.com/Lightning-AI/lightning/pull/15185/commits/8220c0ed1ea30c736f8bd4b6a4290b3378898804), but not everything. I want to keep the PR as focused as possible so it does not distract from what it is meant to do. Happy to receive a PR that updates the rest of "model" occurrences.